pkg/operator/starter.go

```diff
@@ -44,21 +40,11 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
-	dynamicClient, err := dynamic.NewForConfig(ctx.KubeConfig)
-	if err != nil {
-		return err
-	}
 	configClient, err := configv1client.NewForConfig(ctx.KubeConfig)
 	if err != nil {
 		return err
 	}
 
-	v1helpers.EnsureOperatorConfigExists(
```

Comment:
do you not have your own config that you need to ensure exists?

Reply:
Correct, there are no knobs to tweak or parameters to change outside of log level which is set by admin on the ServiceCatalogControllerManager cluster resource.

Reply:
What about the standard "managed/unmanaged/removed" settings to control your operator?

Reply:
That also is set on the ServiceCatalogAPIServer resource:
```
$ oc get ServiceCatalogApiServer cluster -o yaml
apiVersion: operator.openshift.io/v1
kind: ServiceCatalogAPIServer
metadata:
  creationTimestamp: 2019-02-22T13:41:55Z
  generation: 1
  name: cluster
  resourceVersion: "27283"
  selfLink: /apis/operator.openshift.io/v1/servicecatalogapiservers/cluster
  uid: 9e9612e9-36a7-11e9-be39-0e9210288cf4
spec:
  logLevel: Normal
  managementState: Managed
status:
  conditions:
  - lastTransitionTime: 2019-02-22T13:42:07Z
    status: "False"
    type: ResourceSyncControllerFailing
  - lastTransitionTime: 2019-02-22T13:43:04Z
    status: "True"
    type: Available
  - lastTransitionTime: 2019-02-22T13:42:16Z
    status: "False"
    type: Progressing
  - lastTransitionTime: 2019-02-22T13:42:13Z
    status: "False"
    type: WorkloadFailing
.....
```
Initially (at cluster creation time) this resource doesn't exist and the operator is not creating it.  The admin must manually create it and set the managementState to managed to cause the operator to do the install of Service Catalog.  The intention is that while CVO creates the svcat operators, the operand is not installed by default.

Reply:
it's also the object your controller reports its status on (which then the clusteroperator controller bits roll up into the clusteroperator resource)

Reply:
sorry i mean, you should be ensuring this exists instead:
https://github.com/openshift/cluster-svcat-apiserver-operator/blob/master/manifests/02_config.crd.yaml

your defaultconfig object should be an instance of that instead of what it was,right?

Reply:
sorry just read your last paragraph.  makes sense.

Reply:
Thanks for the review @bparees